### DISCUSSION TITLE
compiling ffmpeg to use in wsl but cannot find libdav1d

### DISCUSSION THREAD

**Discussion by james28909** [microsoft/WSL#8513] [General] [answered]
### Version

Microsoft Windows [Version 10.0.22000.675]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

Linux version 5.10.102.1-microsoft-standard-WSL2 (oe-user@oe-host) (x86_64-msft-linux-gcc (GCC) 9.3.0, GNU ld (GNU Binutils) 2.34.0.20200220) #1 SMP Wed Mar 2 00:30:59 UTC 2022

### Distro Version

Ubuntu 20.04

### Other Software

i am wanting to compile ffmpeg for use within wsl itself, and i am following the instructions at https://trac.ffmpeg.org/wiki/CompilationGuide/Ubuntu

when i try to install libdav1d-dev i get the following error:
```
$ sudo apt install libdav1d-dev
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package libdav1d-dev
```

### Repro Steps

have wsl ubuntu 20.04 and try to install libdav1d-dev

### Expected Behavior

i expected to be able to install libdav1d-dev and then attempt compiling ffmpeg 

### Actual Behavior

i cannot get libdav1d-dev installed because apt is not able to find libdav1d-dev

**Comment by elsaco** [comment, marked as answer]
@james28909 if you scroll down to instructions for the `libdav1d` library you'll notice that  `If your repository provides libdav1d-dev, you can install that instead of compiling` and `Otherwise you'll need can build from source`

There are packages in `Debian Sid` you could easily import into `Focal Fossa`. More details at https://packages.debian.org/sid/libdav1d-dev

**Comment by james28909** [comment]
thanks, i was able to build libdav1d from source and finally compile ffmpeg/ffprobe/ffplay from latest source. thanks and sorry for bothering
